_posts/2017/2017-03-10-creating-a-sales-analytics-app-with-ruby-on-rails-2.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Creating a Sales Analytics App with Ruby on Rails, part 2"
+date: "2017-03-10 07:00:00"
+image: blog-header/rails-train-path-straight.jpg
+categories: tech-stories
+authors: ["Unai M."]
+---
+
+A couple of days ago we started writing about how and why we started our Sales analytics app. Today we want to talk about some issues we encountered in our way and how we solved them.
```

Comment:
Last month, I wrote a post about [how and why we started with our Sales Analytics app](https://developer.epages.com/blog/2017/03/02/creating-a-sales-analytics-app-with-ruby-on-rails.html).
In this post, I'd like to talk about some issues we encountered on our way and how we solved them.